contents/newsletter/ai-coding-mistakes.md

```diff
@@ -0,0 +1,171 @@
+---
+title: 7 mistakes to avoid when coding with AI
+date: 2025-09-16
+author:
+ - ian-vanagas
+tags:
+ - AI
+---
+
+No matter how much hype (or money) AI code editors get, engineers are still asking: Do they actually make us better?
+
+At PostHog, the response is mixed. Some find autocomplete great, but need to rip out anything beyond that. Others find they are a huge speedup, but only in specific circumstances.
+
+What is clear for everyone is that coding with AI is a skill. It takes practice and a lot of mistakes to get good at it.
+
+We know this because we've made a bunch of mistakes over the past few years working with these tools, building AI-powered features, and working with companies in the space. We're sharing them here to help you avoid them.
```

Comment:
```suggestion
We know this because we've made a bunch of mistakes over the past few years working with these tools, [building AI-powered features](/newsletter/building-ai-features), and working with companies in the space. Here's what we've learned.
```